CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## v0.7.0-beta (09/09/2019)
+
+**Common**
+
+* Introduced new abstract `olp::thread::TaskScheduler` to be used for async context; provided default thread pool implementation.
+* Added `UserAgent` HTTP header to mark all triggered requests.
+* China Lookup API URLs added (`here-cn` and `here-cn-dev` partitions).
+* Introduced new HTTP abstraction layer `olp::http::Network` without singleton usage. Platform dependent implementations adapted and aligned with coding style.
+* Removed of old `olp::network::Network` with according implementations and helper classes.
```

Comment:
What does `of` refer to? I suggest to remove it.

Reply:
The line is changed to:
* `olp::network::Network` with according implementations and helper classes removed.